modules/web/src/app/wizard/step/cluster/template.html

```diff
@@ -21,7 +21,8 @@
 </ng-container>
 
 
-<form *ngIf="!loadingClusterDefaults" [formGroup]="form">
+<form *ngIf="!loadingClusterDefaults"
+      [formGroup]="form">
```

Comment:
This change is already merged in `main` so please rebase this branch otherwise you'll get merge conflict.